tracing-flame/src/lib.rs

```diff
@@ -235,6 +255,36 @@ where
             out: self.out.clone(),
         }
     }
+
+    /// Configures whether or not periods of time where no spans are entered
+    /// should be included
+    ///
+    /// Defaults to true
+    ///
+    /// Setting this feature to false can help with situations where you have
+    /// large periods of time spent idling or otherwise doing uninteresting work
+    /// where no spans are active. These empty samples can pollute the final
+    /// flamegraph and squish the work you care about into a much smaller
+    /// percentage of the overall graph, making it harder to navigate and in some
+    /// cases even hiding spans.
+    pub fn with_empty_samples(mut self, enabled: bool) -> Self {
+        self.config.empty_samples = enabled;
+        self
+    }
+
+    /// Configures whether or not spans from different threads should be
+    /// collapsed into one pool of events
+    ///
+    /// Defaults to false
+    ///
+    /// Setting this feature to true can help with applications that distribute
+    /// work evenly across many threads. In such cases it can be difficult to get
+    /// an overview of where your application spent most of its time because work
+    /// done in the same span can be split up across many threads.
```

Comment:
Maybe
```suggestion
    /// Setting this feature to true can help with applications that distribute
    /// work evenly across many threads, such as thread pools. In such
    /// cases it can be difficult to get an overview of where the application 
    /// as a whole spent most of its time, because work done in the same
    /// span may be split up across many threads.
```